src/cli/use.js

```diff
@@ -25,7 +25,7 @@ const dev = {
 
 const local = {
   command: 'local',
-  description: 'Use api endpoints set in OS env variables',
+  description: 'Uses api endpoints set in OS env variables.',
```

Comment:
Switches to the local environment, using endpoints set in OS environment variables.